pkgs/development/cuda-modules/cuda/overrides.nix

```diff
@@ -63,7 +77,9 @@ attrsets.filterAttrs (attr: _: (builtins.hasAttr attr prev)) {
     oldAttrs: {
       propagatedBuildInputs = [final.setupCudaHook];
 
-      meta = (oldAttrs.meta or {}) // {
+      allowFHSReferences = true;
+
+      meta = (oldAttrs.meta or { }) // {
```

Comment:
this diff appears irrelevant

Reply:
I'm afraid we'll have more of those, because nixfmt (which we started using with the jetson PR) hasn't stabilized yet